guides/catalog/01 Searching and Rastering Catalog Products.ipynb

```diff
@@ -6,13 +6,10 @@
    "metadata": {},
    "source": [
     "# Searching the Catalog\n",
-    "All objects support the same search interface. `Searches` work by creating a query builder (class `Search`), which can be used in a fluent programming style to refine the search prior to execution by applying filtering, sorting, and limiting of result sets. Normally `Search` objects are created using class methods on one of the primary object types, e.g. `Product.search()`.\n",
     "\n",
-    "The searches are then executed by any of several methods: calling the `count()` method to obtain a count of matching objects, using the `Search` object in an iterating context such as a for loop or a list comprehension to yield each matching object in turn, or calling the `collect()` method which will return a list-like collection object (e.g. `ProductCollection`, `BandCollection`, or `ImageCollection`).\n",
+    "The `Descartes Labs Catalog API` is a single interface through which you can discover existing raster `Products`, search and retrieve their associated `Images`, and manage your own `Products`.\n",
```

Comment:
Nit here and in similar cases: I personally think that modifiers to class names should be avoided or should be left outside of the `code` formatting. For instance:

Only `code`ify the class names:
The Descartes Labs Catalog API is a single interface through which you can discover existing raster `Product`s, search and retrieve their associated `Image`s, and manage your own `Product`s.

Reworded (more elegant, IMO):
The Descartes Labs Catalog API is a single interface through which you can discover existing raster datasets, search and retrieve their associated images, and manage your own datasets.